docs/docs/getting-started/reverse-proxy.md

```diff
@@ -40,32 +39,19 @@ Nginx set up on `proxy` and LibreTime will be installed on `libretime`. You will
 access on both. `libretime` also needs to be able to be accessed from `proxy`
 (`ping libretime` on `proxy`).
 
-On `libretime`, install LibreTime as described in the [install guide](/install). In short
-this means run the following commands:
-
-```
-git clone https://github.com/LibreTime/libretime.git
-cd libretime
-sudo ./install -fiap
-```
-
-Once it has installed, replace `<hostname>localhost</hostname>` in
+On `libretime`, install LibreTime as described in the [install guide](/docs/intro). Once it has installed, replace `<hostname>localhost</hostname>` in
```

Comment:
We don't want to go back to the intro (kind of an entrypoint) when we are already deep in advanced configuration, I would head to the complete install documentation page.